Shooting/Shooting/Scene/GameScene/Player/Object/Missile.h

```diff
@@ -0,0 +1,56 @@
+﻿#pragma once
+
+#include "GameLib/ObjectManager/Object2DBase.h"
+#include "GameLib/DirectX/DirectX.h"
+#include "GameLib/2DHelper/2DHelper.h"
+#include "GameLib/Texture/Texture.h"
+#include <vector>
+
+class Missile :public Object2DBase
+{
+public:
+
+	Missile();
+	~Missile();
+
+	/**
+	* @brief 初期化する関数
+	* @return 初期化が上手くできていないとFALSE
+	*/
+	bool Initialize();
+
+	/**
+	* @brief 解放する関数
+	*/
+	void Release();
+
+	/**
+	* @brief 操作関数
+	*/
+	void Update();
+
+	/**
+	* @brief 描画関数
+	*/
+	void Render();
+
+	/**
+	* @brief ミサイルの生成関数
+	* @param 発射時点での自機のX座標
+	* @param 発射時点での自機のY座標
+	*/
+	void Create(float missile_x, float missile_y);
+
+private:
+	
+	struct MissileInfo
+	{
+		MissileInfo(float missile_x, float missile_y);
```

Comment:
`MissileInfo` がc'torで受け取るxとyはミサイルのxとyであることは自明なので
```suggestion
		MissileInfo(float x, float y);
```
で良いと思いますよ。

Reply:
変数などの名前の変更のコミットで直しました。